### PR TITLE
docs: file descriptor and mysql optimization

### PR DESCRIPTION
**Pull request by chaen**
As promised during the workshop, no need for release notes I guess

### CODE REVIEW

File: docs/source/AdministratorGuide/ServerInstallations/scalingAndLimitations.rst [hunk 1]
```diff
@@ -52,6 +52,16 @@ When you servers are heavily loaded, you may want to tune some kernel parameters
    net.core.somaxconn
    net.core.netdev_max_backlog
 
+You can also adjust the limit of opened files descriptors in the ``Service`` section of the ``/usr/lib/systemd/system/runsvdir-start.service`` file
+
+```
+LimitNOFILE=500000
+```
```
Comment (andresailer) [inline docs/source/AdministratorGuide/ServerInstallations/scalingAndLimitations.rst]:
```suggestion
You can also adjust the limit of opened files descriptors in the ``Service`` section of the ``/usr/lib/systemd/system/runsvdir-start.service`` file::

   LimitNOFILE=500000

```

I don't think markdown works here?

File: docs/source/AdministratorGuide/ServerInstallations/scalingAndLimitations.rst [hunk 2]
```diff
@@ -52,6 +52,16 @@ When you servers are heavily loaded, you may want to tune some kernel parameters
...
+
+Databases
+=========
+
+Every now and then, it is interesting to look at the fragmentation status of your database. This is done by using the ``analyze table`` statement (https://dev.mysql.com/doc/refman/8.4/en/analyze-table.html) possibly followed by the ``optimize table`` statement (https://dev.mysql.com/doc/refman/8.4/en/optimize-table.html).
```
Comment (andresailer) [inline docs/source/AdministratorGuide/ServerInstallations/scalingAndLimitations.rst:63]: I wanted to see if the links actually become links, but the doc creation failed so :shrug:

File: docs/source/AdministratorGuide/ServerInstallations/scalingAndLimitations.rst (hunk 1 above)
Comment (chaen) [reply docs/source/AdministratorGuide/ServerInstallations/scalingAndLimitations.rst]: thanks !

**Review by andresailer — APPROVED** [review]
Looks ok: https://dirac--7736.org.readthedocs.build/en/7736/AdministratorGuide/ServerInstallations/scalingAndLimitations.html#performance-optimisation

File: docs/source/AdministratorGuide/ServerInstallations/scalingAndLimitations.rst (hunk 2 above)
Comment (andresailer) [reply docs/source/AdministratorGuide/ServerInstallations/scalingAndLimitations.rst:63]: Note to posteriority: They do become links.

**Pull request merged**